[program:DJANGO]
user = django
directory=/home/django/app/
command=bash /home/django/app/docker_conf/start_server.sh
autostart=true
autorestart=true
stderr_logfile=/var/log/django.err.log
stdout_logfile=/var/log/django.out.log
stdout_logfile_maxbytes = 10MB
stderr_logfile_maxbytes = 10MB
stopasgroup=true

[program:TWITCHCHATBOT]
directory=/home/django/app/
command=/home/django/.env/bin/python manage.py quotationbot --verbose --ignore
autostart=true
autorestart=true
stderr_logfile=/var/log/chatbot.err.log
stdout_logfile=/var/log/chatbot.out.log
stdout_logfile_maxbytes = 10MB
stderr_logfile_maxbytes = 10MB
stopasgroup=true